client/src/routes/game/+page.svelte

```diff
@@ -30,6 +30,13 @@
   let countdown = GAME_STATE_DURATIONS.WAIT;
   let timer: ReturnType<typeof setInterval>;
 
+  $: if (lobby !== undefined && $user !== null && !lobby.uids.includes($user.uid)) {
+    // then return to join
+    goto(`/join?code=${lobbyCode}`, {
+      replaceState: true,
+    });
+  }
+
```

Comment:
probably want to put this with the other reactive calls towards the bottom starting on `line 118`